website/src/pages/newsletter.js

```diff
@@ -39,64 +39,68 @@ const MyPage = () => {
+                <p>Reste au courant des améliorations apportées au framework Rails et retrouve des tips Ruby</p>
+              </div>
+              <div className='col col--4'>
+                <div className={styles.emoji + ' ' + styles.web}></div><h3>L'actualité web</h3>
+                <p>Une ouverture sur des sujets techs généraux, que ce soit les outils ou les pratiques qui ont fait parler d'eux</p>
+              </div>
+              <div className='col col--4'>
+                <div className={styles.emoji + ' ' + styles.seeding}></div><h3>De l'inclusivité</h3>
+                <p>On valorise la diversité et la représentation de différents profils</p>
+              </div>
             </div>
-            <div class="col col-4 pt-1">
-              <h2 className="hero__title--middle">Ressors informé·e de ta lecture</h2>
-              <p className="m-0">
-                Cette newsletter, c'est pour te donner les outils pour progresser, améliorer ton code et tes pratiques.
-                Elle est là aussi pour t'inspirer avec des projets qui donnent des idées pour te lancer et développer les tiens.
-              </p>
-              <p className="m-0">
-                Le but ce n'est pas que tu dises: "oh la la trop de choses à voir", mais que tu aies <b>les ressources nécessaires pour creuser les sujets qui t'intéressent</b>.
-              </p>
+            <div className='row'>
+              <div className='col col--4'>
+                <div className={styles.emoji + ' ' + styles.monocle}></div><h3>De la sélection</h3>
+                <p>Fini une liste interminable de tutos ou de liens vidéos, à la place quelques lectures clés</p>
+              </div>
+              <div className='col col--4'>
+                <div className={styles.emoji + ' ' + styles.sparkles}></div><h3>De l'inspiration</h3>
+                <p>On partage des projets stylés qui nous rappellent pourquoi on aime coder</p>
+              </div>
+              <div className='col col--4'>
+                <div className={styles.emoji + ' ' + styles.timer}></div><h3>De l'efficacité</h3>
+                <p>Tout ça dans un temps de lecture moyen de 5 minutes</p>
+              </div>
             </div>
+            
           </div>
         </div>
       </div>
 
-      <div className="hero">
-        <div className="container container--centered">
-          <h2 className="hero__title--middle">Lire les précédentes éditions</h2>
-          <p>13 novembre 2020 | <a target="_blank" href="https://womenonrails.substack.com/p/post-mortem-hacktoberfest-women-make-spa-games">👻 Post-mortem, Hacktoberfest, Women Make, l'avenir des SPAs et des jeux</a></p>
+      <div className='hero'>
+        <div className='container container--centered'>
+          <button href='https://womenonrails.substack.com/archive' target='_blank' className={styles.learnMore} >
+            Lire les précédentes newsletters
+          </button>
```

Comment:
Le bouton vers les précédentes newsletters ne s'ouvre pas chez moi :cry:

Reply:
Ah mince tu es sous quel navigateur ? ^^"

Reply:
J'ai testé sous Brave et Firefox (en mode navigation privée), j'ai pas de messages d'erreur dans la console. Mais en remplaçant le tag `<button>` par un tag `<a>` avec  les changements dans la css, le bouton a la même apparance et ça fonctionne :smiley_cat: 
Le `<button>` peut fonctionner comme un lien mais avec un event javascript : https://www.w3docs.com/snippets/html/how-to-create-an-html-button-that-acts-like-a-link.html